components/internal_server/src/healthcheck.py

```diff
@@ -0,0 +1,15 @@
+"""Health check."""
```

Comment:
I guess this code duplication is fine for now, as it will disappear.

Reply:
Agreed.